docs/source/upcoming_release_notes/157-perf_no_hinted_signals.rst

```diff
@@ -0,0 +1,23 @@
+157 perf_no_hinted_signals
+##########################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- removes hinted signal widgets loaded from typhos to improve loadtimes
```

Comment:
```suggestion
- Removes hinted signal widgets loaded from typhos to improve loadtimes
```

Total nitpick, but the docs will look much nicer if these are capitalized appropriately